kura/org.eclipse.kura.linux.watchdog/OSGI-INF/metatype/org.eclipse.kura.watchdog.WatchdogService.xml

```diff
@@ -42,6 +42,17 @@
             required="true"
             default="/dev/watchdog"
             description="Watchdog device."/>
+            
+        <AD id="watchdogAction"
```

Comment:
We have a chance to use a more readable property ``name``. It can be different from ``id`` and the one shown in the Web UI.